### ISSUE TITLE
When joining a query, I must use an 'is'

### ISSUE THREAD

**Issue by lloydtabb** [malloydata/malloy#1360]
In order to perform a join on a query, I must have an is

```
query: current_employees is ...
source: ... {
   join_one: current_employees is current_employees with person_user_name
```

Should be able to be written

```
query: current_employees is ...
source: ... {
   join_one: current_employees with person_user_name
```

**Comment by mtoy-googly-moogly** [comment]
This was fixed in #1388

**Comment by mtoy-googly-moogly** [comment]
Fixed in #1383
